### ISSUE TITLE
Crash when BankID is not installed on device [Android]

### ISSUE THREAD

**Issue by sgrosz** [klarna/kco-mobile-sdk#58]
**Describe the bug**
App crashes when BankId is not installed on the device on Android.

**To Reproduce**
Pay using a payment type which triggers BankID to be opened on device and make sure you don't have bankID installed on the device.

**Expected behavior**
There should be a check to make sure that there is an activity which can start the Intent.

**Device and version (please complete the following information):**
 - Device: HUAWEI MediaPad M5 10.8
 - OS: Android 9
 - SDK Version 1.16.3 & 1.16.4

**Additional context**
```
Fatal Exception: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat=bankid:///?redirect= }
       at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2114)
       at android.app.Instrumentation.execStartActivity(Instrumentation.java:1757)
       at android.app.Activity.startActivityForResult(Activity.java:4803)
       at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:568)
       at android.app.Activity.startActivityForResult(Activity.java:4743)
       at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:554)
       at android.app.Activity.startActivity(Activity.java:5164)
       at android.app.Activity.startActivity(Activity.java:5132)
       at a.b.a.d.k.d.shouldOverrideUrlLoading(d.java:12)
       at android.webkit.WebViewClient.shouldOverrideUrlLoading(WebViewClient.java:77)
       at org.chromium.android_webview.AwContentsClientBridge.shouldOverrideUrlLoading(AwContentsClientBridge.java:17)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next(MessageQueue.java:386)
       at android.os.Looper.loop(Looper.java:175)
       at android.app.ActivityThread.main(ActivityThread.java:7625)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
```
Related file in you SDK (.class file and obfuscated):
```
package a.b.a.d.k;

import a.b.a.b.f;
import a.b.a.d.a;
import android.app.Activity;
import android.content.ActivityNotFoundException;
import android.content.Intent;
import android.net.Uri;
import android.webkit.WebView;
import android.webkit.WebViewClient;

public final class d extends WebViewClient {
    public d(a var1) {
        this.a = var1;
    }

    public boolean shouldOverrideUrlLoading(WebView var1, String var2) {
        if (var2.startsWith("mailto:")) {
            Activity var5 = this.a.i;
            Intent var6;
            if ((var6 = f.a(var2)) != null) {
                try {
                    var5.startActivity(var6);
                } catch (ActivityNotFoundException var3) {
                    var5.runOnUiThread(new a.b.a.d.k.a(var5));
                }
            }

            return true;
        } else if (!var2.startsWith("http")) {
            d var10000 = this;
            Intent var4;
            var4 = new Intent.<init>("android.intent.action.VIEW", Uri.parse(var2));
            var10000.a.i.startActivity(var4);
            return true;
        } else {
            return false;
        }
    }
}
```

**Comment by NMGuner** [comment]
Hi @sgrosz, thank you for pointing out this issue. We will investigate it and include a fix in the following release.

**Comment by NMGuner** [comment]
Hi @sgrosz, we just released a new version(1.6.15) which includes a fix for external apps such as BankID. It would be great if you could try it and inform us whether it is resolved or not on your side.

**Comment by sgrosz** [comment]
Seems to be resolved in 1.6.15
